app/views/pages/consultants.html.erb

```diff
@@ -60,7 +29,7 @@
             <td>
               <ul>
                 <li>Research methodology </li>
-                <li> Quantitative analysis </li>
+                <li>Quantitative analysis </li>
                 <li>Political science/Politics </li>
                 <li>Public opinion </li>
```

Comment:
@kevinreiss I'm approving it. 
I think you should also remove the space at the end of the string. It won't make a difference in the UI from what I see, but why to have extra spaces.
```
<li>Research methodology</li>             
<li>Quantitative analysis</li>
<li>Political science/Politics</li>
<li>Public opinion</li>

Reply:
👍

Reply:
Updated this. I think we can merge this now.